lib/clearance/configuration.rb

```diff
@@ -134,6 +140,7 @@ def initialize
       @secure_cookie = false
       @signed_cookie = false
       @sign_in_guards = []
+      @sign_in_on_password_reset = true
```

Comment:
Is there any preference regarding the order of instance variables and method definitions in this class? I simply added the new configuration option at the end. They seem to be grouped somewhat semantically.